README.ru.md

```diff
@@ -0,0 +1,82 @@
+# Аполлон-11
+[![NASA][1]][2]
+
+:crossed_flags:
+[Bahasa Indonesia][ID],
+[Deutsch][DE],
+[English][EN],
+[Español][ES],
+[Français][FR],
+[Italiano][IT],
+[Português][PT_BR],
+[正體中文][ZH_TW],
+[简体中文][ZH_CN],
+[한국어][KO_KR],
+[हिंदी][HI_IN],
+**Русский**
+
+
+[ID]:README.id.md
+[DE]:README.de.md
+[EN]:README.md
+[ES]:README.es.md
+[IT]:README.it.md
+[FR]:README.fr.md
+[PT_BR]:README.pt_br.md
+[ZH_TW]:README.zh_tw.md
+[ZH_CN]:README.zh_cn.md
+[KO_KR]:README.ko_kr.md
+[HI_IN]:README.hi_in.md
+[RU]:README.ru.md
+
+Оригинальный исходный код бортового управляющего компьютера Аполлон 11 (AGC) для коммандного модуля (Comanche055) и лунного модуля (Luminary099).
+Оцифровано людьми из [Virtual AGC][3] и [MIT Museum][4]. Цель - это создание репозитория с оригинальным исходным кодом миссии Аполлон 11.
+Таким образом приветствуются pull requests (PRs) с исправлениями для любых найденных ошибок в файлах этого репозитория и оригинальных сканах исходного кода для [Luminary 099][5] и [Comanche 055][6], так же как и для любых файлов, которые я мог пропустить.
+
+## Сотрудничество
+Пожалуйста, прочитайте [CONTRIBUTING.md][7] перед тем как открывать pull request.
+
+## Компиляция
+Если вы заинтересованы в компилировании данного исходного кода, смотрите инструкции здесь [Virtual AGC][8].
+
+## Атрибуция
+
+&nbsp;         | &nbsp;
+:------------- | :-----
+Авторское право      | Общественное достояние
+Comanche055    | Часть исходного кода для Colossus 2A, коммандного модуля (CM) бортового управляющего компьютера Аполлон (AGC) для миссии Аполлон 11<br>`Assemble revision 055 of AGC program Comanche by NASA`<br>`2021113-051. 10:28 Апрель. 1, 1969`
+Luminary099    | Часть исходного кода для Luminary 1A, лунного модуля (LM) бортового управляющего компьютера Аполлон (AGC) для миссии Аполлон 11<br>`Assemble revision 001 of AGC program LYM99 by NASA`<br>`2021112-061. 16:27 Июль. 14, 1969`
+Ассемблер      | yaYUL
+Контакты        | Ron Burkey <info@sandroid.org>
+Вебсайт        | www.ibiblio.org/apollo
+Оцифровка | Исходный код был расшифрован или иным способом адаптирован из оцифрованных изображений печатных копий из музея университета MIT. Оцифровку выполнил Paul Fjeld, упорядочил Deborah Douglas из музея. Огромная благодарность обоим.
+
+### Договор и разрешения
+*Получено из [CONTRACT_AND_APPROVALS.agc]*
+
+Эта AGC программа также должна быть упомянута как Colossus 2A.
+
+Эта программа предназначена для использования в командном модуле (CM) как указано в отчете `R-577`. Программа была подготовлена в рамках проекта DSR `55-23870`, при поддержке центра пилотируемых космических кораблей NASA по договору `NAS 9-4065` с участием лаборатории приборостроения, Массачусетского технологического университета (MIT), Кембридж, Массачусетс.
+
+Загружено          | Роль | Дата
```

Comment:
`Представлено` instead of `Загружено` was suggested as being more suitable by the person I asked to review the translation - everything else was :ok_hand: according to them.